rename-files.test.js

```diff
@@ -0,0 +1,75 @@
+const fs = require('fs');
+const { createFiles } = require('./index');
+const { readDirectory, rename, getModifiedTime } = require('./rename-files');
+
+describe('rename functions', () => {
+  beforeEach(done => {
+    createFiles('./new-files', 10, done);
```

Comment:
So in order for this test file to run, a folder called new-files needs to exist first, otherwise this errors out.